src/main/kotlin/app/utils/logging/LoggerUtils.kt

```diff
@@ -189,6 +155,52 @@ fun getDurationInMilliseconds(epochTime: Long): String {
     return elapsedMilliseconds.toString()
 }
 
+class JsonLoggerWrapper(private val wrappedLogger: Logger) {
```

Comment:
This class matches a common pattern - `delegate`. Can we rename `wrappedLogger` to `delegate`?

Reply:
Sure, I'll put that in a following PR.